index.html

```diff
@@ -2355,13 +2355,23 @@ <h3>Unlinkability</h3>
       <section>
         <h3>Selective Disclosure</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are selective
-disclosure digital signature schemes, such as BBS+, that are capable of
-not disclosing correlatable identifiers and ensuring that a different
-but valid digital signature is re-created upon every presentation.
+        <p>
+Selective disclosure is a technique that enables the sender of a pre-signed
+message to reveal parts of the message without revealing the entire message. For
+example, selectively disclosing a digital driver's license for the purposes of
+renting a car might involve the entity presenting the license to only reveal the
+issuing authority, license number, birthday, and authorized motor vehicle class.
+Note that in this case, the license number is correlatable information that is
+being shared but some amount of privacy is still being preserved because the
+driver's full name and address are not being shared.
+        </p>
+
+        <p>
+Selective disclosure is a technique that not all cryptosuites provide. It is
```

Comment:
what do you think about a note mentioning that not only does the creator of the initial message needs to utilize this capability, but the holder (receiver) needs to support it too (in a wallet, or however they may receive it), and also any potential verifiers -- or is this implied?

Reply:
Done in c238acefb5bc4d4340022b6877c0326351f63613.